pages/general/_meta.json

```diff
@@ -1,7 +1,4 @@
 {
   "official_links": "Official links",
-  "expectations": "Testnet expectations",
-  "terms_of_use_and_privacy_policy": "Terms of Use and Privacy Policy",
-  "rate_limits": "Rate Limits",
-  "other_limits": "Other Limits"
+  "terms_of_use_and_privacy_policy": "Terms of Use and Privacy Policy"
```

Comment:
Could you share some reference docs for this JSON, describing the format that we should follow?

Reply:
as far as I can see, the key is the name of the `md` file, the value is the display name. Haven't read the docs

Reply:
Ah, cool, I actually missed the fact that the `terms_of_use_and_privacy_policy` key was there before, I thought you added it now :)